bin/main.rb

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env ruby
+require_relative '../lib/game_logic.rb'
+class Play
+  def names
+    @name = gets.chomp
+    case @name
+    when /\D/
+      puts 'Welcome ' + @name
+    else
+      puts 'Please do not enter numbers or symbols'
+      return self unless @name =~ /^-?[0-9]+$/
+      return self unless @name =~ /^-?[0-9]+$/ || @name == ''
+    end
+  end
+
+  def player1
+    puts 'Hi,Welcome first player, please enter your name'
+    names
+    player1 while @name =~ /^-?[0-9]+$/ || @name == ''
+  end
+
+  def player2
+    puts 'Hi, Welcome second player, please enter your name'
+    names
+    player2 while @name =~ /^-?[0-9]+$/ || @name == ''
+  end
```

Comment:
### Add a player class
- Remove this method from this main rb to a new file called player.rb object class, because you need to add at least two classes and also in the main.rb you don't have to create classes anymore, the main file it's most for call methods